src/rust/consul-connect/Cargo.toml

```diff
@@ -5,7 +5,11 @@ authors = ["Max Wittek <wimax@graplsecurity.com>"]
 edition = "2021"
 
 [dependencies]
-structopt = { version = "0.3", default-features = false }
+clap = { version = "3.0", default_features = false, features = [
```

Comment:
I moved from Structopt to Clap to solve a bug I was having with structopt; I'm going to move the rest of the codebase over to satisfy https://github.com/grapl-security/issue-tracker/issues/868